Tools/TestWebKitAPI/Tests/WebKitCocoa/Download.mm

```diff
@@ -3145,4 +3145,33 @@ HTTPServer childFrameServer({
     Util::run(&checkedDownload);
 }
 
+TEST(WKDownload, DestinationFileAlreadyExists)
+{
+    HTTPServer server { { { "/"_s, { "hi"_s } } } };
+    auto webView = adoptNS([WKWebView new]);
```

Comment:
these all should be `RetainPtr` instead of `auto` right per the smart pointer guidelines, right?